config/database.yml

```diff
@@ -2,13 +2,12 @@ default: &default
   adapter: postgresql
   encoding: unicode
   database: <%= ENV['DB_NAME'] || "c2_#{Rails.env}" %>
-  pool: 5
-
-development: *default
-test: *default
-production:
-  <<: *default
   username: <%= ENV['DB_USERNAME'] %>
   password: <%= ENV['DB_PASSWORD'] %>
   host: <%= ENV['DB_HOST'] %>
   port: <%= ENV['DB_PORT'] %>
```

Comment:
An empty host and port definitely equate to the system defaults for postgres (localhost and.... 5321 -- something like that). I assume the username and password would be the same -- i.e. setting a blank `username` is equivalent to not setting one.

My setup is different than yours, would you mind confirming that a default is needed?